### PR TITLE
Fix git issues

### PR DESCRIPTION
**Pull request by ayshsandu**

### CODE REVIEW

File: en/docs/learn/using-email-address-as-the-username.md [hunk 1]
```diff
@@ -69,22 +69,26 @@
...
     <td><code>               UserNameListFilter              </code></td>
     <td><div class="content-wrapper">
-    <p>Use the mail attribute of the user if neccessary. <strong>LDAP/Active Directory only</strong></p>
+    <p>Use the mail attribute of the user if <strong>necessary. LDAP/Active Directory only</strong> <br/>For example:</p>
     <div class="code panel pdl" style="border-width: 1px;">
     <div class="codeContent panelContent pdl">
-    <pre class="html/xml" data-syntaxhighlighter-params="brush: html/xml; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: html/xml; gutter: false; theme: Confluence"> <code>[user_store] <br> user_name_list_filter=&quot;(&amp;(objectClass=user)(mail=?))&quot;</code>
+    <pre class="html/xml" data-syntaxhighlighter-params="brush: html/xml; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: html/xml; gutter: false; theme: Confluence"> <code>[user_store] <br> user_name_list_filter=`"(&amp;(objectClass=user)(mail=?))"`</code>
     </pre>
+    <div class="admonition info">
```
Comment (gomathyk) [inline en/docs/learn/using-email-address-as-the-username.md]: Please change this also to title

Comment (ayshsandu) [reply en/docs/learn/using-email-address-as-the-username.md]: Fixed

File: en/docs/develop/service-provider-configurations-used-with-apis.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ This section guides you through the configurations you can include in a service
         <td>
             <ul>
                 <li><b>issuer</b> <code>[String]</code> <br>This specifies the issuer. This is the <code>             &lt;saml:Issuer&gt;            </code> element that contains the unique identifier of the service provider. This is also the issuer value specified in the SAML Authentication Request issued by the service provider. When configuring single-sign-on across Carbon servers, ensure that this value is equal to the <code>ServiceProviderID</code> value mentioned in the <code>             &lt;IS_HOME&gt;/repository/conf/deployment.toml</code> file of the relying party's WSO2 Identity Server.</li>
+                <li><b>issuerQualifier</b> <code>[String]</code> <br>
+                    <p>This value is needed only if you have to configure multiple SAML SSO inbound authentication configurations for the same Issuer value. When a Service Provider Qualifier is defined here, it will be appended to the end of the Issuer value when registering the SAML SP in the Identity Server.</p> 
+                    <p>You can configure a number of SAML SPs with the same Issuer and different Service Provider Qualifiers.</p>
```
Comment (YvonneW) [inline en/docs/develop/service-provider-configurations-used-with-apis.md]: When registering a SAML service provider in WSO2 Identity Server, the `service provider qualifier` that is defined here will be appended to the `issuer`.

File: en/docs/develop/service-provider-configurations-used-with-apis.md [hunk 3]
```diff
@@ -18,6 +18,11 @@ This section guides you through the configurations you can include in a service
...
+                    <p>When a Service Provider Qualifier is defined, the issuer of the SAML SSO authentication request is the value specified as the Issuer in the configuration (ex : `travelocity.com` ). The service provider qualifier value should be sent as a query parameter, spQualifier with the HTTP request in the following format.</p>
+                </li>
```
Comment (YvonneW) [inline en/docs/develop/service-provider-configurations-used-with-apis.md]: When a service provider qualifier is defined, the issuer that is defined in the configuration, e.g., `travelocity.com` will be set as the issuer of the SAML SSO authentication request. The `service provider qualifier` should be sent as the query parameter `spQualifier` along with the HTTP request in the following format:

File: en/docs/develop/service-provider-configurations-used-with-apis.md [hunk 4]
```diff
@@ -139,19 +144,13 @@ This section guides you through the configurations you can include in a service
...
                 <li><b>assertionQueryRequestProfileEnabled</b> <code>[Boolean]</code> <br>This is set <code>true</code> to enable Assertion Query Request Profile to query assertions that are persisted to the database when you log in to the service provider application. For more information, see <a href="../../learn/querying-saml-assertions">Querying SAML Assertions</a>.</li>           
+                <li><b>idpEntityIDAlias</strong></b><code>[String]</code><br/>This value can override the value of Identity Provider Entity ID specified under SAML SSO Inbound Authentication configuration in Resident IdP. The Identity Provider Entity ID is used as the issuer of the SAML responses generated from IS. By default, all the SAML responses issued by IS will have the issuer value similar to the Identity Provider Entity ID in Resident IdP’s SAML SSO inbound authentication configuration. But if you want that value to be unique for your SAML SP configuration, you can specify the value here, so that the IdP Entity ID will be overridden with this IdP Entity ID Alias value.</li>
```
Comment (YvonneW) [inline en/docs/develop/service-provider-configurations-used-with-apis.md]: This value can override `identity provider entity Id` that is specified under SAML SSO inbound authentication configuration of the resident identity provider. The Identity Provider Entity Id is used as the issuer of the SAML response that is generated by WSO2 Identity Server. By default, all the SAML responses that are issued by WSO2 Identity Server contain an issuer value that is similar to the `identity provider entity Id` of the resident identity provider's SAML SSO inbound authentication configuration. To make this value unique within the SAML service provider configuration, specify the value here causing the `identity provider entity Id` to be overridden by the `identity provider Id alias`.

File: en/docs/develop/service-provider-configurations-used-with-apis.md (hunk 3 above)
Comment (ayshsandu) [reply en/docs/develop/service-provider-configurations-used-with-apis.md]: Fixed

File: en/docs/develop/service-provider-configurations-used-with-apis.md (hunk 2 above)
Comment (ayshsandu) [reply en/docs/develop/service-provider-configurations-used-with-apis.md]: Fixed

File: en/docs/develop/service-provider-configurations-used-with-apis.md (hunk 4 above)
Comment (ayshsandu) [reply en/docs/develop/service-provider-configurations-used-with-apis.md]: Fixed

**Pull request merged**